### DISCUSSION TITLE
Default Editor + Vue problem

### DISCUSSION THREAD

**Discussion by Ascol57** [GrapesJS/grapesjs#5431] [Q&A]
Hello,
I would like to work with GrapesJs but I copy this code : [https://jsfiddle.net/artur_arseniev/xyofm1qr/](https://jsfiddle.net/artur_arseniev/xyofm1qr/)
I try to start to adapt it for my project but I have an error.

My code I the attached file and Here the error : 
![image](https://github.com/GrapesJS/grapesjs/assets/86880749/89172121-6777-465c-a66d-ac481ac8fca8)
[index.txt (convert to .html)](https://github.com/GrapesJS/grapesjs/files/12763996/index.txt)

Is it an grapesjs error or a Vue error ?

Thanks for helping me
Ascol14

**Comment by artf** [comment]
Check your blocks variable, it's completely make up.
In the example
```js
const blocks = [
  {
    id: 'text',
    label: 'Text',
    category: 'Basic',
    media: svgText,
...
```
your code
```js
const blocks = [
      {
        name: 'Basic',
        list: [ // ?????
          {
            id: 'text',
            label: 'Text',
            category: 'Basic',
            media: svgText,
// ...
```
